std/stdio.d

```diff
@@ -5169,6 +5169,12 @@ Initialize with a message and an error code.
     }
 }
 
+private Error preAllocatedStdioOOMError;
+static this()
```

Comment:
Why is this not `static shared` and why is the error not `immutable`?

Reply:
now using the implementation in core.exception.